clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractHeartbeatRequestManager.java

```diff
@@ -467,12 +441,12 @@ static class HeartbeatRequestState extends RequestState {
         private long heartbeatIntervalMs;
 
         public HeartbeatRequestState(
```

Comment:
could this be just protected now?

Reply:
I think we still need to be able to construct instances for testing.

Reply:
agree, but those tests are in the same package right? so this would be visible to them if protected or package-private like the class itself.